packages/esm-commons-lib/src/components/patient-table/patient-table.scss

```diff
@@ -146,3 +115,16 @@ div#table-tool-bar {
   @include type.type-style('body-compact-01');
   color: $text-02;
 }
+
+.search {
+  max-width: 16rem;
+
+  input {
+    background-color: $ui-02 !important;
+  }
+}
+
+.toolbarContent {
+  height: spacing.$spacing-07;
+  margin-bottom: spacing.$spacing-02;
+}
```

Comment:
Could you add a new line here

Reply:
This will create a space between the seach bar and table which isnt according to the design